src/opm/output/eclipse/EclipseIO.cpp

```diff
@@ -263,7 +263,7 @@ void EclipseIO::writeTimeStep(const SummaryState& st,
     if (!isSubstep) {
         for (const auto& report : schedule.report_config(report_step)) {
             std::stringstream ss;
-            RptIO::write_report(ss, report.first, report.second, schedule, report_step);
+            RptIO::write_report(std::cout, report.first, report.second, schedule, report_step);
```

Comment:
Yeah, no.

Reply:
Relax my friend - this is work in progress

Reply:
> this is work in progress

Then please mark it as such.